test/utilities/http-api-helper.mjs

```diff
@@ -1,49 +1,47 @@
 import axios from 'axios';
 
-class HttpApiHelper {
-    async info(nodeRpcUrl) {
-        return axios({
-            method: 'get',
-            url: `${nodeRpcUrl}/info`,
-        }).catch((e) => {
-            throw Error(`Unable to get info: ${e.message}`);
-        });
-    }
+export async function info(nodeRpcUrl) {
```

Comment:
Is there a reason why you changed this to export methods. We are always exporting the classes so we should do the same here

Reply:
No particular reason, i just had some problems using the exported class. I changed it to a class export now, also did the same thing in steps-utils.mjs.